### PR TITLE
Expandable video commercial template

### PR DESCRIPTION
**Pull request by tijanamiletic**
New template based on the expandable with full-size embedded YouTube video. Works in all resolutions.

On load:
screen shot 2015-08-25 at 16 04 52

Expanded:
screen shot 2015-08-25 at 16 05 08

Test ad available at: /uk/culture?adtest=tm7

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    ExpandableVideo.prototype.create = function () {
+        var videoHeight = this.openedHeight,
+            videoSource = {
+                videoEmbed: (this.params.YoutubeVideoURL !== '') ?
+                    '<iframe id="YTPlayer" width="100%" height="' + videoHeight + '" src="' + this.params.YoutubeVideoURL + '?rel=0&amp;controls=2&amp;fs=0&amp;title=0&amp;byline=0&amp;portrait=0" frameborder="0" class="expandable_video"></iframe>' : ''
+            },
+            $ExpandableVideo = $.create(template(ExpandableVideoTpl, { data: _.merge(this.params, videoSource) }));
+
+        var domPromise = new Promise(function (resolve) {
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video.js]: use one var

File: static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                $ExpandableVideo.appendTo(this.$adSlot);
+                resolve();
+            }.bind(this));
+        }.bind(this));
+
+        bean.on(this.$adSlot[0], 'click', '.ad-exp__open', function () {
+            fastdom.write(function () {
+                this.$ad.css('height', this.openedHeight);
+                $('.slide-video', $(this.$adSlot[0])).css('height', this.openedHeight).addClass('slide-video__expand');
+                var videoSrc = $('#YTPlayer').attr('src') + '&amp;autoplay=1';
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video.js]: use var at the beginning of the function

**Comment by tijanamiletic** [comment]
Suggested changes have been made

File: static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    };
+
+    ExpandableVideo.prototype.create = function () {
+        var videoHeight = this.openedHeight,
+            videoSource = {
+                videoEmbed: (this.params.YoutubeVideoURL !== '') ?
+                    '<iframe id="YTPlayer" width="100%" height="' + videoHeight + '" src="' + this.params.YoutubeVideoURL + '?rel=0&amp;controls=2&amp;fs=0&amp;title=0&amp;byline=0&amp;portrait=0" frameborder="0" class="expandable_video"></iframe>' : ''
+            },
+            $ExpandableVideo = $.create(template(ExpandableVideoTpl, { data: _.merge(this.params, videoSource) })),
+
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video.js]: remove empty line pls

**Comment by Calanthe** [comment]
:+1:

**Pull request merged**